test/config/validation.spec.ts

```diff
@@ -138,21 +138,17 @@ describe('config/validation', () => {
       expect(errors).toMatchSnapshot();
       expect(errors).toHaveLength(0);
     });
-    it('errors for unsafe fileMatches', async () => {
+
+    it('validates regEx for each fileMatch', async () => {
       const config = {
-        npm: {
-          fileMatch: ['abc ([a-z]+) ([a-z]+))'],
-        },
-        docker: {
-          fileMatch: ['(x+x+)+y'],
-        },
+        fileMatch: ['js', '***$}{]]['],
       };
       const { warnings, errors } = await configValidation.validateConfig(
-        config
+        config,
+        true
       );
       expect(warnings).toHaveLength(0);
-      expect(errors).toHaveLength(2);
-      expect(errors).toMatchSnapshot();
```

Comment:
@nihalanisumit This snapshot test should be readded

Reply:
@nihal9ns these tests need to be added back

Reply:
since we are not using safe-regex, it won't throw error for unsafe regex, that's why I removed this test. @rarkins @ViceIce